recipes/freyja/meta.yaml

```diff
@@ -42,7 +42,7 @@ requirements:
     - pysam
     - biopython
     - seaborn-base
-    - pyarrow
+    - pyarrow >=10.0.1
```

Comment:
The effect of this is the same as bumping the python requirement to `python >=3.10`, which is in line with the [installation recommendations of the upstream project](https://github.com/andersen-lab/Freyja/tree/main?tab=readme-ov-file#installation-via-conda). So updating the python requirement seems like the more natural thing to do to me.

Reply:
yeah, not entirely sure myself what exactly is going on in this dependency jungle :sweat_smile:
What's clear is that pyarrow >=10.0.1 requires python >=3.8 (so your change is equivalent to dropping python 3.7 silently. Conversely, requiring python >=3.8 with the current freyja build on the command line helps conda over its resolver issue (makes it install python 3.11
and pyarrow 11.0.0).
My original python >=3.10 suggestion results in exactly the same env as python >=3.8, and I thought that might be good because it's  the version *recommended* by upstream, but yes maybe that would be too aggressive.

My conclusion would then be to *only* update to `python >=3.8` (which is more explicit then `pyarrow >=10.0.1`, which can never yield a py37 build anyway).

Reply:
shouldn't matter much if you *also* require `pyarrow >=10.0.1`.

Reply:
Great! Thanks for your help! 
I've bumped python version, and left pyarrow version just in case, it shouldn't matter as you say, but just to be extra safe as this is a weird issue per se.